### ISSUE TITLE
How to guarantee that just a .exe in a git repo is open-source?

### ISSUE THREAD

**Issue by DaniellMesquita** [jceminer/cn_gpu_miner#30]

**Comment by platinum4** [comment]
Yeah this is kind of a shitty way of putting out experimental builds.  At least zip them so Windows Defender is not shredding these before they get downloaded.

**Comment by jceminer** [comment]
For the same reason Claymore is not open source.

**Comment by DaniellMesquita** [comment]
> For the same reason Claymore is not open source.

And what is the reason that Claymore isn't open source?
